scripts/build_config.py

```diff
@@ -0,0 +1,83 @@
+import os
+import sys
+import uuid
+import random
+import logging
+import configparser
+from hpfeeds.add_user import create_user
+from cifsdk.exceptions import AuthError
+from cifsdk.client.http import HTTP as Client
+
+
+def main():
+    logging.info("Running build_config.py")
+    MONGODB_HOST = os.environ.get("MONGODB_HOST", "mongodb")
+    MONGODB_PORT = os.environ.get("MONGODB_PORT", "27017")
+    HPFEEDS_HOST = os.environ.get("HPFEEDS_HOST", "hpfeeds3")
+    HPFEEDS_PORT = os.environ.get("HPFEEDS_PORT", "10000")
+    IDENT = os.environ.get("IDENT", "")
+    SECRET = os.environ.get("SECRET", "")
+    CHANNELS = os.environ.get("CHANNELS", "amun.events,conpot.events,thug.events,beeswarm.hive,dionaea.capture,dionaea.connections,thug.files,beeswarm.feeder,cuckoo.analysis,kippo.sessions,cowrie.sessions,glastopf.events,glastopf.files,mwbinary.dionaea.sensorunique,snort.alerts,wordpot.events,p0f.events,suricata.events,shockpot.events,elastichoney.events,rdphoney.sessions,uhp.events")
+    CIF_HOST = os.environ.get("CIF_HOST", "https://public-cif-stingar.security.duke.edu")
```

Comment:
suggest we remove this default so there are not unexpected impacts to CIF side

Reply:
Ahh, yeah. Definitely. I'll add that change.

Reply:
CIF_HOST default has been removed.